src/components/Card/Card.js

```diff
@@ -7,6 +7,12 @@ function Card(props) {
             <Statistic type="Breed" details={props.name}/>
             <Statistic type="Height" details={props.height}/>
             <Statistic type="Temperament" details={props.temperament}/>
+            <button className="like-button" onClick={()=>{
+                props.clickEvent(props.id)
+            }}>
+                <span
+                className="heart-icon">&#10084;</span> Like
```

Comment:
Not sure that we need this on a separate line?

Reply:
done